src/Components/News/Detail/NewsDetailLayout.js

```diff
@@ -14,29 +16,24 @@ const NewsImage = lazy(
 );
 
 const NewsDetailLayout = () => {
-  const [newData, setNewData] = useState({});
   const [newsDescription, setNewsDescription] = useState("");
-  const [loading, setIsLoading] = useState(true);
+  const [loading, setIsLoading] = useState(false);
```

Comment:
why you are changing this to false? with this change it will never show the loader

Reply:
Oh no! I did it in some test just to check a few things and I forgot like this! sorry! I fix it!

Reply:
Sara, in the final PR the loading was fine, in true. Apparently I returned it to true in some other commit.